src/core_modules/capture-core/storageControllers/mainStorageController.ts

```diff
@@ -17,7 +17,7 @@ const computeServerCacheVersion = ({ minor, patch, tag }) =>
     computeTagVersionPart(tag);
 
 const computeAppCacheVersion = () => {
-    const appCacheVersionAsString = process.env.REACT_APP_CACHE_VERSION;
+    const appCacheVersionAsString = process.env.DHIS2_CACHE_VERSION;
```

Comment:
Should this be `import.meta.env.DHIS2_CACHE_VERSION` or is the app-shell doing some magic here?